src/python/WMCore/DataStructs/Run.py

```diff
@@ -1,58 +1,46 @@
 #!/usr/bin/env python
-# pylint: disable=W0104
+
 """
 _Run_
 
 container representing a run, and its constituent lumi sections
-
 """
 
-
-
 from WMCore.DataStructs.WMObject import WMObject
 
+
 class Run(WMObject):
     """
     _Run_
 
     Run container, is a list of lumi sections
-
     """
-    def __init__(self, runNumber = None, *newLumis):
+
+    def __init__(self, runNumber=None, *newLumis):
         WMObject.__init__(self)
         self.run = runNumber
-        self.lumis = []
-        self.lumis.extend(newLumis)
+        self.eventsPerLumi = []
+        self.eventsPerLumi.extend(newLumis)
 
     def __str__(self):
-        return "Run%s:%s" % (self.run, list(self.lumis))
-
+        return "Run%s:%s" % (self.run, list(self.eventsPerLumi))
 
     def __lt__(self, rhs):
         if self.run != rhs.run:
             return self.run < rhs.run
-        return list(self.lumis) < list(rhs.lumis)
+        return list(self.eventsPerLumi) < list(rhs.lumis)
```

Comment:
Eric, this may become quite expensive operations since you create copies of existing lists. If you'll deal with lots of runs and their lumis your code may slow down substantially. This comment applies to all __op__ methods and properties you define.